apis/nucleus/src/components/ActionsToolbarItem.jsx

```diff
@@ -19,6 +19,7 @@ const Item = React.forwardRef(({ item, addAnchor = false }, ref) => {
       onClick={item.action}
       disabled={disabled}
       style={style}
+      className="njs-cell-action"
```

Comment:
Should this be exposed in the spec as well?

Reply:
yes it should